cl/corpus_importer/utils.py

```diff
@@ -107,6 +107,18 @@ async def mark_ia_upload_needed(d: Docket, save_docket: bool) -> None:
         await d.asave()
 
 
+def is_appellate_court(court_id: str) -> bool:
```

Comment:
I think it'd be a good idea to create an async version of this method as well. This way, it can be used in async methods without relying on `sync_to_async`, allowing us to benefit from the advantages of native async methods.

Reply:
What would be an appropriate name for this method? Would prefixing it with 'a' be suitable?

Reply:
`ais_appellate_court` is pretty bad, but is the best I can think of. It will always be prefixed by `await` anyway, so maybe that's enough for people to figure out that `ais` means `async is`? Can you guys do better?

Reply:
Yeah! I think adding the `a` prefix is a good idea, following Django conventions for sync and async methods.

Reply:
what about `has_appellate_jurisdiction()` and `ahas_appellate_jurisdiction()`?

Reply:
Doesn't feel right either. That feels like a different legal question.

Reply:
I found `aexists` to be pretty awkward at first too. Maybe `ais` is fine?